app/utils/categories.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {gte, lt} from 'semver';
+
+import {Client4} from '@client/rest';
+import {getConfig} from '@mm-redux/selectors/entities/general';
+import {getNewSidebarPreference} from '@mm-redux/selectors/entities/preferences';
+import {GlobalState} from '@mm-redux/types/store';
+
+export const shouldShowLegacySidebar = (state: GlobalState) => {
+    const config = getConfig(state);
+    const serverVersion = config.Version || Client4.getServerVersion();
+
+    // No server version? Default to legacy.
+    if (!serverVersion) {
+        return true;
+    }
+
+    // Older servers default to Legacy unless experimental flag is set
+    if (lt(serverVersion, '5.32.0')) {
```

Comment:
Why not use `isMinimumServerVersion` like in the rest of the codebase?

Reply:
I needed less than; I think `isMinimumServerVersion` is less than or equal - and knowing my luck, v5.31.9999999 would come knocking on this statement...